connector/src/test/scala/com/google/cloud/spark/bigquery/it/SparkBigQueryEndToEndITSuite.scala

```diff
@@ -474,6 +480,18 @@ class SparkBigQueryEndToEndITSuite extends FunSuite
     assert(df.schema == allTypesTable.schema)
   }
 
+  /* test("write all types to bq - json format") {
+    writeToBigQuery(allTypesTable, SaveMode.Overwrite, "json")
```

Comment:
@davidrabinowitz  Currently this test fails because the ordering of data read from big query in json format is different from how it was loaded. We can discuss this further

Reply:
That's why we need to add the schema as well.